### PR TITLE
feat: update readme

### PR DESCRIPTION
**Pull request by brettgriffin-md**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,23 @@
 
 pg_duckdb is a Postgres extension that embeds DuckDB's columnar-vectorized analytics engine and features into Postgres. We recommend using pg_duckdb to build high performance analytics and data-intensive applications.
 
+Third party developers will be able to extend both DuckDB and the PostgreSQL integration by writing their own DuckDB extensions. Those extensions will be distributed through the DuckDB community extension repository.
+
 pg_duckdb was developed in collaboration with our partners, [Hydra](https://hydra.so) and [MotherDuck](https://motherduck.com).
 
+
+## Roadmap
```
Comment (jtigani) [inline README.md]: There is already a roadmap section below. Maybe this should be goals?

File: README.md [hunk 2]
```diff
@@ -2,8 +2,23 @@
...
+
+* Create and manage analytical tables with good performance on analytical queries
+* Broad support for Postgres types and functions within analytical context
+* Query external datasets on S3, GCP and Azure (Parquet, Iceberg and Delta Lake)
+* Join native Postgres and analytical data efficiently
+* Seamlessly run queries read and write queries against local data and MotherDuck tables
```
Comment (jtigani) [inline README.md]: there is an extra 'queries' here. You might rephrase it as
Seamlessly run read and write queries against native postgres and cloud tables stored in MotherDuck.

File: README.md [hunk 3]
```diff
@@ -2,8 +2,23 @@
...
+* Allow for easy configuration and deployment of DuckDB extensions
```
Comment (jtigani) [inline README.md]: This should be connected to a user goal. Maybe add "to enable additional functionality from the DuckDB ecosystem"

File: README.md [hunk 4]
```diff
@@ -2,8 +2,23 @@
...
+* Columnar indexing of native Postgres tables
```
Comment (jtigani) [inline README.md]: I don't quite understand this one, I'd cut it.

File: README.md [hunk 5]
```diff
@@ -2,8 +2,23 @@
...
+* Real-time synchronization of analytical and transactional data
```
Comment (jtigani) [inline README.md]: I don't quite know what this means or how we'd do it, I'd cut it.

File: README.md [hunk 6]
```diff
@@ -2,8 +2,23 @@
...
+* Creation and maintenance of materialized views across table types
```
Comment (jtigani) [inline README.md]: I don't know how important this is, I'd cut it.

File: README.md (hunk 1 above)
Comment (jtigani) [inline README.md]: We're missing something about storage. Suggest the bullet point from [here](https://docs.google.com/document/d/1RHUva6Qm8w_cRpfalCTMXI1b7boDsu4SHSxGPAW6LGg/edit) 
Allow storing data in a columnar format using DuckDBs native files

Comment (jtigani) [inline README.md]: Not a bad idea to synchronize the other goals from here
https://docs.google.com/document/d/1RHUva6Qm8w_cRpfalCTMXI1b7boDsu4SHSxGPAW6LGg/edit#bookmark=id.kp7rsuuaev2f

File: README.md [hunk 7]
```diff
@@ -2,8 +2,23 @@
 
 pg_duckdb is a Postgres extension that embeds DuckDB's columnar-vectorized analytics engine and features into Postgres. We recommend using pg_duckdb to build high performance analytics and data-intensive applications.
 
+Third party developers will be able to extend both DuckDB and the PostgreSQL integration by writing their own DuckDB extensions. Those extensions will be distributed through the DuckDB community extension repository.
```
Comment (jtigani) [inline README.md:5]: I think it is good to call out our other partners, Neon and Microsoft either here or in the contributing section below.

File: README.md (hunk 4 above)
Comment (brettgriffin-md) [reply README.md]: Let me get clarification Heikki before I cut this (this section is a combination of his roadmap (based on a list from Boaz) and our Goals section.

File: README.md (hunk 5 above)
Comment (brettgriffin-md) [reply README.md]: If this means near-time replication of transactional data to an analytical table, I think it should be kept. A big limitation of the extension as it is defined today is that it expects users to recreate their analytical table from the transactional table. I believe this suggests a capability of synchronizing mutations in the transactional table with the derived analytical table.

Comment (brettgriffin-md) [reply README.md]: Going to strike this and add back if/when I get clarification on functionality.

File: README.md (hunk 4 above)
Comment (brettgriffin-md) [reply README.md]: Going to strike this and add back if/when I get clarification on functionality.

**Pull request merged**